src/main/java/com/zendesk/maxwell/RowMap.java

```diff
@@ -157,6 +157,24 @@ public String pkAsConcatString() {
 		return keys;
 	}
 
+	public String buildPartitionKey(List<String> partitionColumns) {
+		if (partitionColumns.isEmpty()) {
+			return table;
+		}
+
+		String partitionKey="";
+		for (String pc : partitionColumns) {
+			Object pcValue = null;
+			if (data.containsKey(pc))
+				pcValue = data.get(pc);
+			if (pcValue != null)
+				partitionKey += pcValue.toString();
+		}
+		if (partitionKey.isEmpty())
+			return table;
+		return partitionKey;
+	}
```

Comment:
should add some documentation somewhere around what the key will be (how various columns end up formatted, etc) -- I've found it's often necessary to re-code partition-key methods in other languages.  I was debating whether or not to separate the values with `:` -- I don't think there's much value there.